src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetails.java

```diff
@@ -82,7 +84,11 @@ public boolean extractQuestionDetails(
 
         String forceUnevenDistributionString =
                 HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMDISTRIBUTEUNEVENLY);
+                                                        Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMDISTRIBUTEUNEVENLY);
```

Comment:
Same here.

Reply:
Do you mean to do something like this:

```
String forceUnevenDistributionString = HttpRequestHelper.getValueFromParamMap(requestParameters,
        Const.ParamsNames.FEEBBACK_QUESTION_CONSTSUMDISTRIBUTEUNEVENLY);
```

The reason why I did that was because of the code that was already present prior to my PR:

![image](https://user-images.githubusercontent.com/24493463/38535923-ddbf6956-3cb8-11e8-9a52-131d83ea5eef.png)

But I can change all of them if this is better coding style.

Reply:
Yup. Oh, I see. Yes; can change all of them.